zebrad/src/commands/connect.rs

```diff
@@ -98,32 +98,31 @@ impl ConnectCmd {
 
 type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
 
-struct Connect<ZN, ZNF, ZS> {
+struct Connect<ZN, ZS>
+where
+    ZN: Service<zebra_network::Request>,
```

Comment:
any reason to have just a partial bound here rather than the full bound or no bound?

Reply:
ah, `ZN: Service` so that `ZN::Future` is available below.